cats/src/test/scala/magnolify/cats/test/TestEq.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package magnolify.cats.test
+
+import cats.Eq
+import magnolify.cats.semiauto.EqDerivation
+import magnolify.test.ADT._
+import magnolify.test.JavaEnums
+import magnolify.test.Simple._
+import magnolify.shared.UnsafeEnum
+
+import java.net.URI
+import java.time._
+
+object TestEq {
+
+  // other
```

Comment:
I think this comment doesn't add clarity/context so early in the file :D maybe we should move the "simple" section above this "other" section so that "other" makes more sense in comparison